app/assets/javascripts/application.js

```diff
@@ -20,8 +20,6 @@
 //= require leaflet-providers/leaflet-providers.js
 //= require leaflet-toolbar/dist/leaflet.toolbar.js
 //= require leaflet-distortableimage/dist/leaflet.distortableimage.js
-//= require leaflet-easybutton/src/easy-button.js
-//= require leaflet-google/index.js
```

Comment:
Hey @icarito @jywarren @alaxalves, this removal is causing trouble loading the map in development environment. Also, any reason for this removal?

Reply:
I had to remove those since Yarn wasn't being able to load those dependencies. Have you readded those and verified if if still fails on production and development?